app/assets/stylesheets/arclight/bootstrap_overrides.scss

```diff
@@ -4,10 +4,18 @@
 
 .nav.nav-tabs {
   margin-bottom: 0;
+  flex-wrap: nowrap;
   & .nav-item .nav-link:not(.active) {
     background: $gray-100;
     border-color: $nav-tabs-border-color;
   }
+  & .nav-item .nav-link {
+    height: 100%;
+  }
+  & .nav-item {
+      overflow-wrap: break-word;
+      width: 25%;
```

Comment:
Could this 25% be removed? It seems like the `flex-fill` class correctly parcels out the width for each tab that's available. Noting as well that #645 shows we're moving toward having fewer than four tabs in some views (e.g., components won't have an overview tab & files won't have online content tab).